grails-app/services/org/pih/warehouse/importer/ProductAssociationImportDataService.groovy

```diff
@@ -48,18 +45,18 @@ class ProductAssociationDataService {
                 if (params.hasMutualAssociation || it.hasMutualAssociation) {
                     return  (
                             params['product'] == it['product'] &&
-                            params['associatedProduct'] == it['associatedProduct'] &&
-                            params['code'] == it['code']
-                        ) || (
+                                    params['associatedProduct'] == it['associatedProduct'] &&
```

Comment:
is it only github going crazy, or some additional indents came in? 
+ a nitpicky comment that we should add into our coding conventions and just reminded myself about it  - the `&&`/`||` should be placed at the beginning of a line, not at the end.